GPU/Common/DrawEngineCommon.h

```diff
@@ -30,10 +30,14 @@
 
 class VertexDecoder;
 
+// int maxVerts = DECODED_VERTEX_BUFFER_SIZE / 2 / vertexSize; =~ 54000
+// 65536 * 6 * 2
```

Comment:
I think we could delete this part, I think it's just for the old values.

-[Unknown]